WGLMakie/test/runtests.jl

```diff
@@ -19,19 +19,12 @@ import Electron
 end
 
 excludes = Set([
-    "image scatter",
-    # missing transparency & image
-    "Image on Surface Sphere",
-    # Marker size seems wrong in some occasions:
-    "Hbox",
-    "UnicodeMarker",
-    # Not sure, looks pretty similar to me! Maybe blend mode?
-    "Test heatmap + image overlap",
+    "Image on Surface Sphere", # TODO: texture rotated 180°
     # "heatmaps & surface", # TODO: fix direct NaN -> nancolor conversion
```

Comment:
Does this comment need to be included here?

Reply:
They still don't handle nan_colors the same as GLMakie. Iirc that causes this test to be different